index.html

```diff
@@ -1642,6 +1642,15 @@ <h2 id="OfflineAudioContext">
             </dl>
           </dd>
           <dt>
+            readonly attribute unsigned long length
```

Comment:
Since the parameter is called numberOfFrames, we should probably call this numberOfFrames too instead of length.